src/cms/static/js/pages/page_mirroring.js

```diff
@@ -22,12 +22,13 @@ function render_mirrored_page_field(event){
             throw new Error(response.statusText);
         }).then(function (html) {
             // Load rendered field into div and show it
-            u('#mirrored_page_div').html(html);
+            u('#mirrored_page_div').empty();
+            u('#mirrored_page_div').append(html);
```

Comment:
```suggestion
            u('#mirrored_page_div').html(html);
```
This essentially equivalent, so I propose to keep the simpler form of it and mark the warning as false positive.
Since the html is a direct response from our server, I don't see a security threat here.